app/assets/stylesheets/custom.scss

```
@@ -0,0 +1,106 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
+@import "colors";
+
+/* universal */
+
```

```suggestion
/* universal */
```
coding convention thì duwois line comment ko có line trống nhé bạn nhỏ